app/featureflags/feature-flags-public/src/main/kotlin/com/hedvig/android/featureflags/HedvigUnleashClient.kt

```diff
@@ -70,14 +79,17 @@ class HedvigUnleashClient(
       .build()
   }
 
-  private fun createContext(market: String, appVersion: String) = UnleashContext.newBuilder()
-    .appName(APP_NAME)
-    .properties(
-      mutableMapOf(
-        "appVersion" to appVersion,
-        "appName" to APP_NAME,
-        "market" to market,
-      ),
-    )
-    .build()
+  private fun createContext(market: String, appVersion: String, memberId: String): UnleashContext {
```

Comment:
You could pass a `String?` memberId here and pass it as a property if its non-null.